docs/web/community/ecosystem.md

```diff
@@ -0,0 +1,21 @@
+# Ecosystem
+
+## Backends
+
+### [pandas](https://github.com/pandas-dev/pandas)
+
+[pandas](https://github.com/pandas-dev/pandas) is a Python package that provides fast, flexible, and expressive data structures designed to make working with "relational" or "labeled" data both easy and intuitive. It aims to be the fundamental high-level building block for doing practical, real world data analysis in Python. Additionally, it has the broader goal of becoming the most powerful and flexible open source data analysis / manipulation tool available in any language. It is already well on its way towards this goal.
+
+## Related Packages / Dependencies
+
+### [SQLAlchemy](https://github.com/sqlalchemy/sqlalchemy)
+
+[SQLAlchemy](https://github.com/sqlalchemy/sqlalchemy) is the Python SQL toolkit and Object Relational Mapper that gives application developers the full power and flexibility of SQL. SQLAlchemy provides a full suite of well known enterprise-level persistence patterns, designed for efficient and high-performing database access, adapted into a simple and Pythonic domain language.
+
+
+## Packages that extend Ibis
+
+### [sql_to_ibis](https://github.com/zbrookle/sql_to_ibis)
+
+[sql_to_ibis](https://github.com/zbrookle/sql_to_ibis) is a Python package that translates SQL syntax into ibis expressions. This provides the capability of using only one SQL dialect to target many different backends 
```

Comment:
```suggestion
[sql_to_ibis](https://github.com/zbrookle/sql_to_ibis) is a Python package that translates SQL syntax into ibis expressions. This provides the capability of using only one SQL dialect to target many different backends.
```
The last sentence could be clearer. I would explain a bit more in detail. And I think it would be useful to also add a simple code example.